src/ngx_http_lua_phase.c

```diff
@@ -107,4 +107,22 @@ ngx_http_lua_inject_phase_api(lua_State *L)
     lua_setfield(L, -2, "get_phase");
 }
 
+
+#ifndef NGX_LUA_NO_FFI_API
+int
+ngx_http_lua_ffi_get_phase(ngx_http_request_t *r, char **err)
+{
+	ngx_http_lua_ctx_t  *ctx;
+
+	ctx = ngx_http_get_module_ctx(r, ngx_http_lua_module);
+	if (ctx == NULL) {
+		*err = "no request context";
+		return NGX_ERROR;
```

Comment:
And yeah, you should really use the `ngx-releng` tool to check your code's coding style to avoid common issues.